### ISSUE TITLE
Permissions wrong after brew cask install (~/.vscode-oss)

### ISSUE THREAD

**Issue by sdaitzman** [VSCodium/vscodium#389]
**Describe the bug**
I am unable to install packages. When I click "install" on a package in the sidebar, a dialog pops up in the bottom right that says, for example, "Failed to install 'pkief.material-icon-theme'."

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Go to extensions sidebar
2. Search for any extension
3. Click install

**Expected behavior**
I expected the package to install

**Screenshots**
<img width="481" alt="Screen Shot 2020-04-23 at 9 18 59 AM" src="https://user-images.githubusercontent.com/1499476/80103791-79251d80-8543-11ea-844f-32c04598acca.png">

**Desktop (please complete the following information):**
<img width="698" alt="Screen Shot 2020-04-23 at 9 19 17 AM" src="https://user-images.githubusercontent.com/1499476/80103844-93f79200-8543-11ea-99b7-480c27b1c52e.png">
<img width="410" alt="Screen Shot 2020-04-23 at 9 20 24 AM" src="https://user-images.githubusercontent.com/1499476/80103886-ab367f80-8543-11ea-963c-7fdfd4936a15.png">

**Additional context**
Let me know if any debug logs would be helpful :~)

**Comment by sdaitzman** [comment]
I was just able to fix this after running a `chown` of the package directory, but I'm unsure what caused the permissions to be incorrect in the first place.

I ran:

```bash
sudo chown -R myusername ~/.vscode-oss/extensions/
```

**Comment by fahid-aiond** [comment]
@sdaitzman Thanks for the answer. Fixed.

I installed the VScodium through `brew cask install vscodium` command as suggested in the website.

I had few files with root user ownership in this folder `~/.vscode-oss/extensions/`,

When I tried to watch all the installed extensions then the main file causing the issue was `~/.vscode-oss/extensions/.obselete` again with the root user ownership.

with your command, I change the ownership to my user and it worked.
